docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+= Creating a group snapshot
+
+In {ProductName}, the integration service provides `group` snapshot to allow users to submit multiple related pull/merge requests under one application to their repos, have them tested together and have the CI results posted back to pull/merge requests.
```

Comment:
Does the integration service disable the creation of non-group snapshots?

How does the system handle updating the ITS runs for successive group snapshots (i.e. what happens when group/component1's build finishes the group/component2's build finishes)?

Reply:
Integration service won't disable non-group snapshot. Component snapshots work as before.
Integration tests running for component snapshot work as before. Group snapshot give users one chance to run some integration test for multiple pull/merge request in one snapshot together. [Here](https://docs.google.com/document/d/1LnS_iv3IuOunFdz9c0VcGd0S7xbVCxsS4mmc-rv3xyo/edit#heading=h.76hl0cvwxpfc) is the spike doc.